### PR TITLE
Basic functionality

### PR DESCRIPTION
**Pull request by kerymov**

### CODE REVIEW

File: Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt [hunk 1]
```diff
@@ -38,27 +37,51 @@ import androidx.compose.ui.unit.dp
...
+}
+
+@ExperimentalFoundationApi
 @Composable
 fun GameScreen() {
+    var credits by remember {
+        mutableStateOf(1000)
+    }
+    val elementsInRow = 3
+    val slotsItems = listOf(
```
Comment (yev-kanivets) [inline Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt:58]: `slotItems` -> usually, there is just one plural in the variable name ;)

File: Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt [hunk 2]
```diff
@@ -68,15 +91,18 @@ fun GameScreen() {
...
+            Credits(credits)
+            Slots(slots, elementsInRow, result)
+            Spin(
+                spin = {
+                    slots.forEachIndexed { index, _ ->
+                        slots[index] = slotsItems[Random.nextInt(slotsItems.lastIndex)]
+                    }
+                    result = compare(slots)
+                    credits += if (result) Random.nextInt(maxWin - minWin) + minWin
+                    else -100
```
Comment (yev-kanivets) [inline Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt:103]: No need to break the line here.

File: Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt [hunk 3]
```diff
@@ -126,51 +152,46 @@ fun Credits(
...
-            LazyRow(
-                horizontalArrangement = Arrangement.Center,
-                verticalAlignment = Alignment.CenterVertically
+        items(slots.size) {
+            Box(
+                modifier = Modifier
+                    .padding(5.dp)
+                    .clip(RoundedCornerShape(10.dp))
+                    .background(
+                        if (result) Green50
```
Comment (yev-kanivets) [inline Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt:173]: No need to break a line for `if/else` without `{}`.

**Review by yev-kanivets — APPROVED** [review]
Wow, great work 👍 Code looks well (NP for Box, no need for buttons).

A few extra challenges:
- make the app full screen
- adapt to the landscape
- 3 rows (diagonal, vertical, horizontal wins)

File: Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt (hunk 1 above)
Comment (kerymov) [reply Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt:58]: Oh yes, exactly)

File: Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt (hunk 2 above)
Comment (kerymov) [reply Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt:103]: Ok, thanks

File: Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt (hunk 3 above)
Comment (kerymov) [reply Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt:173]: Ok

Comment (kerymov) [reply Slots App/app/src/main/java/com/example/challengeslotsapp/GameScreen.kt:173]: I'll fix it in the next commit

**Pull request merged**